web/src/user_status_ui.ts

```diff
@@ -48,6 +131,28 @@ export function open_user_status_modal(): void {
             input_field().trigger("focus");
         },
     });
+
+    render_scheduled_end_time_widget();
+
+    $("#scheduled-end-time-description").text(no_scheduled_end_time_set_message);
+    if (scheduled_end_time) {
+        $("#scheduled_end_time_widget .dropdown_widget_value").text(
+            $t(
+                {defaultMessage: "{N}"},
```

Comment:
I don't understand what the translators are being asked to translate here; is it the curly braces?